lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -297,6 +309,19 @@ void EnforceABIImpl::run() {
   OpaquePC->addFnAttr(Attribute::NoUnwind);
   OpaquePC->addFnAttr(Attribute::ReadOnly);
 
+  RaiseExc = M.getFunction("raise_exception_helper");
+  revng_assert(RaiseExc != nullptr);
+
+  // Create a bitmask to trace which CSVs need to be saved for the function
+  auto *ConstantZero = ConstantInt::get(Type::getInt64Ty(Context), 0);
+  new GlobalVariable(M,
+                     Type::getInt64Ty(Context),
+                     false,
+                     GlobalValue::ExternalLinkage,
+                     ConstantZero,
+                     "RestoreCSVBitMask");
+  RestoreCSVBitMask = M.getGlobalVariable("RestoreCSVBitMask");
```

Comment:
Why this extra lookup? Just get the pointer from the previous statement.

Reply:
Because originally I declared that variable in `support.c`. Must have added that code at a later time.